example/lib/main.dart

```diff
@@ -68,6 +68,7 @@ class FlutterDeckExample extends StatelessWidget {
         TitleSlide(),
         LayoutStructureSlide(),
         BlankSlide(),
+        FooterSlide(),
```

Comment:
I don't think this is the right place for the slide since I would like to keep all the template slides one after another. Maybe move the footer slide just before the `HiddenSlide`?